script.js

```diff
@@ -0,0 +1,42 @@
+const inputBox = document.getElementById("name");
+const inputBox2=document.getElementById("email");
+const submitBtn = document.querySelector(".form button");
+const users=document.querySelector(".users");
+const url=' https://reqres.in/api/users';
+let result="";
+let c=0;
+
+var array=[]
+inputBox.onkeyup=()=>{
+    array[0]=inputBox.value;
+}
+
+inputBox2.onkeyup=()=>{
+    array[1]=inputBox2.value;
+    if(array[0]!="" && array[1] !="" && array.length===2)
+    {
+        submitBtn.classList.add("active"); 
+    }
+}
+
+
+fetch(url)
+  .then(res => res.json())
+  .then(res =>{
+      console.log(res.data)
```

Comment:
remove unwanted comments or console statements when pushing final code

- [ ] remove console and comment